js/screen-saver.js

```diff
@@ -0,0 +1,46 @@
+var app = app || {};
+
+app.screenSaver = (function() {
+
+	'use strict';
+
+	return {
+
+		screenSaverActive: false,
+
+		initialize: function() {
+
+			this.createTimer();
+
+			$(document).click(_.bind(function(){
+
+				clearTimeout(this.mouseTimeOut);
+
+				if (this.screenSaverActive) {
+					this.hideScreenSaver();
+				}
+
+				this.createTimer();
+	
+			}, this));
+		},
+
+		createTimer: function() {
+			this.mouseTimeOut = setTimeout(_.bind(function(){
+				this.showScreenSaver();
+			}, this), app.config.screenSaver.idleTime);
+		},
+
+		showScreenSaver: function() {
+			$('html').removeClass('loaded');
```

Comment:
Can we add another class to the `<html>` element instead of using the `.loaded` class? To make it more semantic (meaningful).